packaging/leapp.spec

```diff
@@ -138,7 +136,9 @@ BuildRequires:  python3-setuptools
 BuildRequires:  python3-pytest-cov
 %endif
 
-Requires: leapp-framework-dependencies = 1
+Requires: python3-setuptools
```

Comment:
Just to make it clear - We don't need the deps here because those are already defined as Python3, the whole problem we originally had was python-something or something-python vs python2-something this does not account for python3-something since they already used the right naming.

Reply:
@vinzenz I don't agree. That'S part of the problem. Another issue - that is not there know, but can be in future - is about changes of dependencies between original & target system. Nowhere is written, that you will not need additional dependency in the target system, e.g. because the python3-somethin package will be split into several other packages. Another point is, that this is against our documentation (not sure whether already both  parts of doc are merged) as well. We declared, that all outside dependencies should be defined instide the deps metapackage. I think we really should not care about whether the problem is relevant now for Python3. Just keep the logic as it was supposed to be used originally - or improve it. But this is basically wrong change. - the same for findutils and the others. These are *outer dependencies*.

Reply:
@vinzenz in case you want another discussion, I prefer call and then we can write here result of the discussion :)